### DISCUSSION TITLE
on  v1.118.0 PHP has no colors anymore

### DISCUSSION THREAD

**Discussion by linpi1** [pulsar-edit/.github#273] [Community Package Help]
with the new version  v1.118.0 and on a php file, there are no colors anymore. 
I didn't find any settings about that. 
There is something to do or it is a little bug ?

**Comment by linpi1** [comment]
great, on the rolling release this problem has been already fixed :-)

**Reply by florian-strasser** [reply]
I can't confirm that, for me most php files still look like this:
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/c8d73ea2-6442-4e21-a352-d942bffc7515">
